.eslintrc

```diff
@@ -28,6 +28,7 @@
     "react/no-multi-comp": [2, {"ignoreStateless": true}],
     "react/no-unused-prop-types": 2,
     "react/no-unused-state": 2,
+    "react/prop-types": 0,
```

Comment:
prop-types are useful for runtime check (for debugging purpose) on the props. 
Typescript interface are just at compile-time. 
I think that would be useful to add prop-types as well